cypress/e2e/cloud/pinned.test.ts

```diff
@@ -0,0 +1,330 @@
+import {Organization} from '../../../src/types'
+import {createFirstTask} from '../shared/tasks.test'
+
+describe('Pinned Items', () => {
+  let orgID: string
+  beforeEach(() => {
+    cy.flush()
+
+    cy.signin().then(() =>
+      cy.fixture('routes').then(({orgs}) => {
+        cy.get('@org').then(({id}: any) => {
+          orgID = id
+          cy.setFeatureFlags({
+            pinnedItems: true,
+            docSearchWidget: true,
+          }).then(() => {
+            cy.wait(1000)
```

Comment:
remove this, please! The issue here is that you're setting the feature flags and then navigating to a different page. We should try to flip the order. Or use the nav bar to navigate to the new page. See this: https://influxdb.atlassian.net/wiki/spaces/PB/pages/2367356935/Cypress#Feature-Flags

Reply:
AH thank you. I was wondering why i needed to add this back, apologies!